src/crimson/os/seastore/lba_manager/btree/lba_btree_node_impl.cc

```diff
@@ -646,7 +650,7 @@ LBALeafNode::find_hole_ret LBALeafNode::find_hole(
       min = i->get_key() + i->get_val().len;
     }
   }
-  if (min + len <= max) {
+  if (min + len <= max && min != end) {
```

Comment:
min == end here should never happen.

Reply:
please see the output log, it really happen.

Reply:
Oh, I know.  I mean that the chain of events that caused it is a bug.